client/reducers/placeReducer.js

```diff
@@ -27,9 +27,19 @@ const initialState = {
 };
 const placeReducer = handleActions(
   {
-    [actionTypes.SET_DETAILS](state, action) {
+    /**
+     * I'm a hesitant to use this not adding a details
+     * property in the state creating one at setplace action type
+     */
+    [types.SET_DETAILS](state, action) {
       return { ...state, ...action.payload };
     },
+    [types.SET_PLACE](state, { payload }) {
```

Comment:
i understand, but you can still try reusing `set_details`, in that way, we can keep the data as flat as possible.

Reply:
Will use the set_deatils and submit it before 12nn.